### ISSUE TITLE
Privacy request from firefox add-on contain wrong domain

### ISSUE THREAD

**Issue by Mannshoch** [languagetool-org/languagetool#4577]
I use a local LanguageTool server but the add-on wish I have to accept data transfer to languagetool.org.
https://languagetool.org/webextension/install

Because I do not use languagetool.org in my case, if you wish to have me approve data transfer it should request it for my own server.

- [ ] add this website (https://languagetool.org/webextension/install) to the LanguageTool server 
- [ ] modify the LanguageTool add-on that way it requests https://languagetool.org/webextension/install from the local server
e.g.: localhost/v2/webextension/install

**Comment by danielnaber** [comment]
The dialog you're referring to is only shown on installation, isn't it? At that time, we have to assume the default configuration, and that's the one which points to languagetool.org.

**Comment by Mannshoch** [comment]
Not really, It appeared suddenly. May its cause because I didn't use the browser on this machine since a week and Firefox did an Update earlier. This dialog appears if I click on the ! which appear in every textbox at moment.

**Comment by danielnaber** [comment]
> This dialog appears if I click on the ! which appear in every textbox at moment.

You mean it keeps appearing again, even after you've confirmed the dialog (i.e. clicked "Ok")? That shouldn't be the case. Does it help to re-install the add-on?

**Comment by Mannshoch** [comment]
No, I do not confirm the dialog until now. The problem is:

> I do not use languagetool.org in my case, if you wish to have me approve data transfer it should request it for my own server.

**Comment by Mannshoch** [comment]
Yes I know that. I do not confirm it to be able to may attach new information to this ticket if you need.
Why did you close it?

**Comment by Mannshoch** [comment]
What do you think about my proposal? I have rewritten it a bit so it is may better understandable.

> include this website (https://languagetool.org/webextension/install) into the LanguageTool server
> modify the LanguageTool add-on that way it requests webextension/install from the server it is connected
e.g.: {myserver-orIP}/webextension/install

**Comment by danielnaber** [comment]
The dialog should only appear on the very first installation, i.e. at a time when there's no user-specific server set yet.

**Comment by Mannshoch** [comment]
You said that earlier but my proposal is to made it more generic so it works also for local server installation. So it will ask If I really wish to use this NAS or IP as server.

And because I still have this dialog even I had changed the server is also strange. 
In the current situation, at that moment I change the server it should not any more appear.

**Comment by etlweather** [comment]
@danielnaber What if I am running this off-line? The privacy dialog page on your web site can't be reached because the computer is not connected to the internet at all. How to bypass that?

**Comment by danielnaber** [comment]
> What if I am running this off-line?

Sorry, offline use is not supported.
